components/ImagesGallery/ImagesGallery.vue

```diff
@@ -445,6 +448,21 @@ export default {
             ...Array.from(scicrunchData['abi-plot'], plot => {
               const id = plot.identifier
               const file_path = plot.dataset.path
+              const thumbnail = this.getThumbnailForScaffold(
```

Comment:
Maybe this function could be named `getThumbnailForScicrunchItem`? 

And then `getThumbnailsForPlot` could be deleted.

Reply:
Good catch, they work differently and plot items should use the one for plot instead.

Reply:
I like those method names.